<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossPanel</class>
 <widget class="QWidget" name="BossPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>116</y>
     <width>140</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>BOSS NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>9</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">TIME</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="active_title_label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>46</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ACTIVE BOSS NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="active_time_label">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>82</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ACTIVE TIME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
